django/example/settings.py

```diff
@@ -9,7 +9,7 @@
 
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['0.0.0.0']
```

Comment:
Maybe. Still, it doesn't work on Mac by default  without them

Reply:
It depends on a way you spawn and access your project. If you spawn it like `python manage.py runserver` and access it as `127.0.0.1:8000` - it will surely work.
But, in case I am wrong - ok, let's merge it. I don't mind.